### DISCUSSION TITLE
Bug Report

### DISCUSSION THREAD

**Discussion by DEVILRANG** [FakeFishGames/Barotrauma#14505] [Bug reports]
### Disclaimers

- [x] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

1.船员排序在每个回合会恢复默认排序。
2.在 海达尔裂隙前哨战的营救联盟人质任务中，只出现两个人质，解开手铐并且清空敌人也不会显示完成任务。

地图种子：xSWRPTNu

### Reproduction steps

正常游玩

### Bug prevalence

Happens every now and then

### Single player or multiplayer?

Single player

### Version

v1.6.101.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
I'm afraid there's insufficient information here for us to address the issue.
